lib/ansible/modules/cloud/ovh/ovh_monthly_billing.py

```diff
@@ -0,0 +1,128 @@
+    - Enable monthly billing on OVH cloud intances (be aware OVH does not allow to disable it).
+requirements: [ "ovh" ]
+options:
+    project_id:
+        required: true
+        type: str
+        description:
+            - ID of the project, get it with https://api.ovh.com/console/#/cloud/project#GET
+    instance_id:
+        required: true
+        type: str
+        description:
+            - ID of the instance, get it with https://api.ovh.com/console/#/cloud/project/%7BserviceName%7D/instance#GET
+'''
+
+EXAMPLES = '''
+# basic
+  - ovh_monthly_billing: project_id=0c727a20aa144485b70c44dee9123b46 instance_id=8fa89ad2-8f08-4220-9fa4-9695ea23e948
+
+# more complex
+  - os_server_facts:
+      cloud: myProjectName
+      region_name: myRegionName
+      server: myServerName
+    # force run even in check_mode
+    check_mode: no
+
+  - name: ovh_monthly_billing
+    ovh_monthly_billing:
+      project_id: "{{ openstack_servers.0.tenant_id }}"
+      instance_id: "{{ openstack_servers.0.id }}"
+'''
+
+RETURN = '''
+'''
+
+import os
+import sys
+
+try:
+    import ovh
+    import ovh.exceptions
+    from ovh.exceptions import APIError
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            project_id=dict(required=True),
+            instance_id=dict(required=True),
+        ),
+        supports_check_mode=True
+    )
+
+    # Get parameters
+    project_id = module.params.get('project_id')
+    instance_id = module.params.get('instance_id')
+    project = ""
+    instance = ""
+    result = ""
+
+    # Connect to OVH API
+    client = ovh.Client()
+
+    if not HAS_OVH:
+        module.fail_json(msg='python-ovh is required to run this module, see https://github.com/ovh/python-ovh')
+
+    # Check that the instance exists
+    try:
+        project = client.get('/cloud/project/{0}'.format(project_id))
+    except ovh.exceptions.ResourceNotFoundError:
+        module.fail_json(msg='project {0} does not exist'.format(project_id))
+
+    # Check that the instance exists
+    try:
+        instance = client.get('/cloud/project/{0}/instance/{1}'.format(project_id, instance_id))
+    except ovh.exceptions.ResourceNotFoundError:
+        module.fail_json(msg='instance {0} does not exist in project {1}'.format(instance_id, project_id))
+
+    # Is monthlyBilling already enabled or pending ?
+    if instance['monthlyBilling'] is not None:
+        if instance['monthlyBilling']['status'] in ['ok', 'activationPending']:
+            module.exit_json(changed=False, result=instance['monthlyBilling'])
+
+    if module.check_mode:
+        module.exit_json(changed=True, msg="Dry Run!")
+
+    else:
+        try:
+            result = client.post('/cloud/project/{0}/instance/{1}/activeMonthlyBilling'.format(project_id, instance_id))
+            module.exit_json(changed=True, result=result['monthlyBilling'])
```

Comment:
change this `result` key to something useful such as `ovh_billing_status` or `ovh_billing_info`